packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -12,9 +12,8 @@ flutter:
         pluginClass: UrlLauncherPlugin
       ios:
         pluginClass: FLTURLLauncherPlugin
-      # TODO(mvanbeusekom): Temporary disabled until web is migrated to nnbd (advised by @blasten).
-      #web:
-      #  default_package: url_launcher_web
+      web:
+        default_package: url_launcher_web
```

Comment:
Remove this, until `integration_test` from sdk lands on master, and all plugins can be migrated to that!

Reply:
(This is currently breaking CHANNEL:stable tests, as expected)

Reply:
Reverted, but STABLE is not failing because of this.